### PR TITLE
define SCD1,2,3

### PR DESCRIPTION
**Pull request by atbida**

### CODE REVIEW

File: docs/low-code-spark/gems/source-target/file/delta.md [hunk 1]
```diff
@@ -228,11 +228,13 @@ object writeDelta {
 
 ## Delta MERGE
 
-### Upsert data with Delta
+Delta merge lets us take advantage of three types of slowly changing dimension(SCD) approaches: SCD1, SCD2, and SCD3. These approaches refer to record modifications in which history is not retained (SCD1), history is retained at the row level (SCD2), or history is retained at the column level (SCD3).
```
Comment (slyons) [inline docs/low-code-spark/gems/source-target/file/delta.md:231]: (Nitpick) It's not the only use case for Delta MERGE, but it's the mechanism that allows us to do SCD.

File: docs/low-code-spark/gems/source-target/catalog-table/delta.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ tags:
 Reads and writes Delta tables that are managed by the execution environment's Metadata catalog (Metastore).
 
 :::note
-Please choose set the property `provider` to `Delta` on the properties page.
+Please set the property `provider` to `Delta` on the properties page.
```
Comment (slyons) [inline docs/low-code-spark/gems/source-target/catalog-table/delta.md:15]: Good catch

File: docs/low-code-spark/gems/source-target/catalog-table/delta.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ For time travel on Delta tables:
 
 1. Only `Read Timestamp` **_OR_** `Read Version` can be selected, not both.
 2. Timestamp should be between the first commit timestamp and the latest commit timestamp in the table.
-3. Version needs to be an integer. Its value has to be between min and max version of table.
+3. Version needs to be an integer with value between min and max version of table.
```
Comment (slyons) [inline docs/low-code-spark/gems/source-target/catalog-table/delta.md:36]: Good catch

**Review by slyons — APPROVED** [review]

File: docs/low-code-spark/gems/source-target/file/delta.md [hunk 4]
```diff
@@ -530,7 +530,7 @@ object writeDeltaSCD2 {
 
 ### SCD3
 
-Using the same customer tables as in our merge example above, output and configurations for SCD3 merge will look like below.
+Using the same customer tables as in our merge example above, output and configurations for SCD3 merge will look like below. Let's track change for `customer_zip_code` by adding a column to show the previous value.
```
Comment (atbida) [inline docs/low-code-spark/gems/source-target/file/delta.md:533]: why must we add a column? shouldn't Delta be capturing this history in the log? @slyons

Comment (atbida) [reply docs/low-code-spark/gems/source-target/file/delta.md:533]: Maybe the delta log is what enables us to capture these previous values?
image

**Pull request merged**